init.lua

```diff
@@ -60,6 +60,29 @@ local source = string.sub(debug.getinfo(1,'S').source, 2)
 local path = string.sub(source, 1, string.find(source, "/[^/]*$"))
 local noicon = path .. "noicon.png"
 
+local function cycle(altTabTable, altTabIndex, dir)
```

Comment:
Looks like you moved the function up. Please revert this change and focus on your enhancement.